mvc/controllers/articles.controller.js

```diff
@@ -24,4 +24,19 @@ exports.getComments = (req, res, next) => {
         const comments = result
         res.status(200).send({ comments });
     }).catch(next);
+}
+
+exports.addComment = (req, res, next) => {
+    const id = req.params.article_id;
+    const username = req.body.username;
+    const comment = req.body.body;
+
+    if (!id || !username || !comment) {
+        return res.status(400).send({ msg: 'Bad request' })
+    }
+
```

Comment:
This is a really nice check to see but this can and should be handled in the model file and should ideally be using a custom error handler for this as well.